demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -41,26 +57,36 @@
  * This demo uses the Mosquitto test server. This is a public MQTT server; do not
  * publish anything sensitive to this server.
  */
-#define SERVER                 "test.mosquitto.org"
+#define BROKER_ENDPOINT           "test.mosquitto.org"
+
+/**
+ * @brief Length of MQTT server host name.
+ */
+#define BROKER_ENDPOINT_LENGTH    ( ( uint16_t ) ( sizeof( BROKER_ENDPOINT ) - 1 ) )
 
 /**
  * @brief MQTT server port number.
  *
  * In general, port 8883 is for secured MQTT connections.
  */
-#define PORT                   8883
+#define BROKER_PORT               ( 8883 )
 
 /**
  * @brief Path of the file containing the server's root CA certificate.
  *
  * This certificate should be PEM-encoded.
  */
-#define SERVER_CERT            "mosquitto.org.crt"
+#define SERVER_CERT_PATH          "certificates/mosquitto.org.crt"
+
+/**
+ * @brief Length of path to server certificate.
+ */
+#define SERVER_CERT_PATH_LEN      ( ( uint16_t ) ( sizeof( SERVER_CERT_PATH ) - 1 ) )
```

Comment:
This one ends with _LEN while the others end with _LENGTH

Reply:
yea. Thanks. fixed.